src/SFA.DAS.AdminService.Web/Startup.cs

```diff
@@ -106,51 +105,83 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
+
+            services.AddTransient<ISessionService>(x =>
+                new SessionService(x.GetService<IHttpContextAccessor>(), Configuration["EnvironmentName"]));
+
+            services.AddTransient<CertificateDateViewModelValidator,CertificateDateViewModelValidator>();
+
+            services.AddTransient<IOrganisationsApiClient>(x =>
+                    new OrganisationsApiClient(ApplicationConfiguration.ClientApiAuthentication.ApiBaseAddress,
+                        x.GetService<ITokenService>(), 
+                        x.GetService<ILogger<OrganisationsApiClient>>()));
+
+            services.AddTransient<IContactsApiClient>(x => new ContactsApiClient(
+                ApplicationConfiguration.ClientApiAuthentication.ApiBaseAddress,
+                x.GetService<ITokenService>(), 
+                x.GetService<ILogger<ContactsApiClient>>(),
+                x.GetService<IContactApplyClient>()));
+            
+            services.AddTransient<IApplyApiClient>(x => new ApplyApiClient(
+                ApplicationConfiguration.ApplyApiAuthentication.ApiBaseAddress,
+                x.GetService<ILogger<ApplyApiClient>>(),
+                x.GetService<IApplyTokenService>()));
+
+            services.AddTransient<IApiClient>(x => new ApiClient(
+                ApplicationConfiguration.ClientApiAuthentication.ApiBaseAddress,
+                x.GetService<ILogger<ApiClient>>(),
+                x.GetService<ITokenService>()));
+
+            services.AddTransient<IContactApplyClient>(x => new ContactApplyClient(
+                ApplicationConfiguration.ApplyApiAuthentication.ApiBaseAddress, 
+                x.GetService<ITokenService>(), 
+                x.GetService<ILogger<ContactApplyClient>>()));
+
+            services.AddTransient<IValidationService, ValidationService>();
+            services.AddTransient<IAssessorValidationService, AssessorValidationService>();
+            services.AddTransient<ISpecialCharacterCleanserService, SpecialCharacterCleanserService>();
+            
+            services.AddTransient<IAssessmentOrgsApiClient>(x =>
+                new AssessmentOrgsApiClient(ApplicationConfiguration.AssessmentOrgsApiClientBaseUrl));
+
+            services.AddTransient<IIfaStandardsApiClient>(x =>
+                new IfaStandardsApiClient(ApplicationConfiguration.IfaApiClientBaseUrl));
+
+            services.AddTransient<IAzureTokenService, AzureTokenService>();
+
+            services.AddTransient<IAzureApiClient>(x => new AzureApiClient(
+                ApplicationConfiguration.AzureApiAuthentication.ApiBaseAddress,
+                x.GetService<IAzureTokenService>(),
+                x.GetService<ILogger<AzureApiClientBase>>(),
+                x.GetService<IWebConfiguration>(),
+                x.GetService<IOrganisationsApiClient>(),
+                x.GetService<IContactsApiClient>()));
+
+            services.AddTransient<CacheService, CacheService>();
```

Comment:
You should be able to do `services.AddTransient<CacheService>();` for these types of registrations.

Reply:
Have updated to the simplified versions